mlir/lib/ExecutionEngine/ArmSMEStubs.cpp

```diff
@@ -18,6 +33,10 @@
 // a path to an alternate implementation.
 
 extern "C" {
+void MLIR_ARNSMEABISTUBS_EXPORT _mlir_arm_sme_abi_stubs_exportstub() {
+  // TODO: is function is not used, need at least one exported symbol on Windows
+  // to generate export library.
+}
```

Comment:
Rather than add this dummy function, I think you can just add `MLIR_ARNSMEABISTUBS_EXPORT` to one (or all) of the functions already here.

Reply:
This makes sense, although I can't test it immediately on non-windows, does those tests run on GH precommit runners?

Reply:
done

Reply:
They're not run as part of pre-commit, but this looks like it should be an NFC on Linux. I can give this patch a quick test tomorrow.

Reply:
ok, thanks